### DISCUSSION TITLE
More options for the /ignor command

### DISCUSSION THREAD

**Discussion by Augmented-Bersy** [Zrips/CMI#5561] [Feature Requests]
I have very light rules on the server about chat. And my players has option to ignore people who insult them.
So I would be glad if some of my suggestions will be noticed

**1.** Add option to set timer for ignore, so players can ignore someone in the middle of cross-fire. They want to interact with that player(s) out of their conflicts 

/ignore [player] [time]

**2.** Add option to notice player if someone whos he ignore is talking in chat right now. Maybe message in a chat, maybe with a delay. 

"[Player] is talking in chat, but you ignore him"

**3.** Addition to prev suggestion. Connect ignore to TAG system. 

"[Player] is trying to tag you, but you ignore him"

**There can be some permissions or toggle commands for this options.** 

Thank you for your work and sorry for my english

**Comment by mrfloris** [comment]
Please visit https://github.com/Zrips/CMI/discussions to post suggestions
